web/src/pages/code-of-conduct.js

```diff
@@ -80,7 +80,7 @@ const CodeOfConduct = ({ location }) => {
               please notify a QueerJS organizer as soon as possible.
             </p>
             <p>
-              If you can’t find the organizer, reach out to {organizer.name} at
+              If you can’t find the organizer, reach out to {organizer.name} at{' '}
```

Comment:
fwiw this looks super ugly but i've done it countless times in my react apps haha think it's the best way to go